src/constants.ts

```diff
@@ -1,2 +1,3 @@
 export const minutesInDay = 1440;
 export const minutesInHour = 60;
+export const TOAST_ERROR_DURATION = 50_000;
```

Comment:
It was `5000` before, why now `50000`?

Reply:
Dang, was a debugging edit I have forgotten. Fixed in 
https://github.com/MORE-Platform/more-studymanager-frontend/pull/585/commits/0c5262b49e8689891c7598209d62bcbb7d30674e